app/renderer/js/main.js

```diff
@@ -87,7 +90,12 @@ class ServerManagerView {
 			showNotification: true,
 			betaUpdate: false,
 			silent: false,
-			lastActiveTab: 0
+			lastActiveTab: 0,
+			dnd: false,
+			oldSettings: {
+				showNotification: true,
```

Comment:
I think this should inherit from the main settings, if it's not null. Something like this - 
```javascript
oldSettings: {
	showNotification: ConfigUtil.getConfigItem('showNotification') || true,
	silent: ConfigUtil.getConfigItem('silent') || false
}

```

Reply:
What are the advantages?

I think it should be like this.
```
oldSettings: {
	showNotification: (ConfigUtil.getConfigItem('showNotification') === null) || true,
	silent: (ConfigUtil.getConfigItem('silent') === null) || false
}
```